README-Scotch.md

```diff
@@ -52,17 +53,39 @@ In the following examples it is assumed that one mpi rank is mapped to each Cosm
 | 4               | 32                  | tleaf 3 4 1000 2 10 4 5 |
 | 8               | 64                  | tleaf 3 8 1000 2 10 4 5 |
 
-The user needs to define this tleaf structure and save it as `target.tgt` in the directory they will run SWIFT from. Ongoing work focuses on automatically generating this target architecture upon run time. 
+The first index denotes the number of layers of the tleaf structure and the subsequent index pairs are the number of nodes in a layer and the cost to communicate between them. For example the numbers represented in the 64 MPI rank case (`tleaf 3 8 1000 2 10 4 5`) are as follows: reading right to left the `3` denotes the number of layers in the leaf structure, `8` denotes the number of vertices in the first layer (which corresponds to Cosma 8 compute nodes), `1000` the relative cost for internode communication, `2` deontes the number of vertices in the second layer (number of sockets on each cosma 8 node), `10` relative cost for intersocket communication, `4` denotes the number of vertices in the final layer (number of NUMA regions per socket on cosma 8) and finally `5` is the relative cost of intrasocket communication.  
 
+The user needs to define this tleaf structure and save it as `target.tgt` in the directory they will run SWIFT from. Ongoing work focuses on automatically generating this target architecture upon run time. 
 
 With OpenMPI the `mpirun` option `--map-by numa` has been found to be optimal. This is in contrast to previously suggested `--bind-to none` on the cosma8 [site](https://www.dur.ac.uk/icc/cosma/support/cosma8/).
 
 Scotch details
 ----------------
 
-Scotch carries out the mapping using various strategies which are outlined in the documentation. The Scotch strategy is passed to the Mapping functions. For the runs carried out here it was found that the global flag `SCOTCH_STRATBALANCE` and a imbalance ratio of `0.05` worked best. These values are passed to `SCOTCH_stratGraphMapBuild`. 
+Scotch carries out the mapping using various strategies which are outlined in the [documentation](https://gitlab.inria.fr/scotch/scotch/-/blob/master/doc/scotch_user7.0.pdf), a list of the strategies trialed include: `SCOTCH_STRATDEFAULT`, `SCOTCH_STRATBALANCE`, `SCOTCH_STRATQUALITY` and `SCOTCH_STRATSPEED`. The Scotch strategy string is passed to the Mapping functions. For the runs carried out here it was found that the global flag `SCOTCH_STRATBALANCE` and an imbalance ratio of `0.05` worked best. These values are passed to [`SCOTCH_stratGraphMapBuild`](https://github.com/UCL/swiftsim/blob/cb06b0e5c3d8457c474d0084d973f437d29b20d8/src/partition.c#L1657). 
+
+One issue with Scotch is that when the number of mpi ranks is comparable to the dimensionality of the modelled SWIFT system the optimal mapping strategy doesn't neccessarily map to all available NUMA regions. At present this isn't handled explicity in the code and the paritition reverts to a vectorised or previous partitioning.
```

Comment:
```suggestion
One issue with Scotch is that when the number of mpi ranks is comparable to the dimensionality of the modelled SWIFT system the optimal mapping strategy doesn't necessarily map to all available NUMA regions. At present this isn't handled explicitly in the code and the partition reverts to a vectorised or previous partitioning.
```

Reply:
What does "the partition reverts to a vectorised or previous partitioning." mean? Or would this be obvious if I read the Scotch docs more carefully? :flushed:

Reply:
This is the fall back partitioning used by SWIFT, see [here](https://github.com/UCL/swiftsim/blob/be7d0f1ceef6de3a94017751f8fb8b58ff19614e/src/partition.c#L2491C2-L2493C62).